cg/meta/transfer/lims.py

```diff
@@ -52,15 +53,24 @@ def __init__(self, status: Store, lims: LimsAPI):
         }
 
     def transfer_samples(
-        self, status_type: SampleState, include: str = "unset", sample_id: str = None
+        self,
+        status_type: SampleState,
+        include: str = "unset",
+        order_date_cutoff: datetime | None = None,
+        sample_id: str = None,
     ):
         """Transfer information about samples."""
 
         if sample_id:
             samples: list[Sample] = self.status.get_samples_by_internal_id(internal_id=sample_id)
         else:
-            samples: list[Sample] = self._get_samples_to_include(include, status_type)
+            samples: list[Sample] = self._get_samples_to_include(
+                include=include, status_type=status_type
+            )
 
+        if order_date_cutoff:
+            LOG.info(f"Remove samples ordered before {str(order_date_cutoff)}")
+            self._remove_older_orders(samples=samples, order_date_cutoff=order_date_cutoff)
```

Comment:
If you want to perform the removal based on the existence of the cutoff, it would be nice to finish the execution of the function there, so that we don't do extra logic that does nothing afterwards
```suggestion
        if order_date_cutoff:
            LOG.info(f"Remove samples ordered before {str(order_date_cutoff)}")
            self._remove_older_orders(samples=samples, order_date_cutoff=order_date_cutoff)
            return
```